hack/update-gofmt

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+# Copyright 2019 The Kubernetes Authors.
```

Comment:
fix the copyright year and comment applies for all the other copyright headers as well

Reply:
@mkumatag Should I make it as 2022?

Reply:
The copyright headers should contain the year at which it is created right?

Reply:
I changed the year to 2022

Reply:
yes, 2022